public/assets/js/vendor/bootstrap.min.js

```diff
@@ -0,0 +1,7 @@
+/*!
```

Comment:
Why is this file being committed,when it is also being installed as a dependency in `package.json`?

It would be better if to just choose one option:

- Commit this file --> Nothing else needs to be done
- Install via package.json --> Add a `postinstall` run script to copy this file across from within `node_modules`, and `.gitignore` this one

Reply:
I removed the npm dependecy.